index.js

```diff
@@ -0,0 +1,6 @@
+const button = document.querySelector('.menu-bars');
+const mobilenav = document.getElementById('mobile-nav');
+
+button.onclick = function () {
+  mobilenav.style.display = 'block';
+};
```

Comment:
You did a great job implementing the display menu functionality 👏 , but Kindly add functionality to close the menu when the user clicks on the close button or one of the links because it is included in the requirement:
![close](https://user-images.githubusercontent.com/95412943/210754448-01f2fe15-7198-426f-a1cd-e1711bc0036f.PNG)
![link](https://user-images.githubusercontent.com/95412943/210754466-8841a76b-9cba-4ca7-8f16-88c1bdcbb0dc.PNG)

- You can watch this video to see the issue:
  

https://user-images.githubusercontent.com/95412943/210754630-7bfd41f3-8f79-44f2-8e1b-3b74cb972a79.mp4

Reply:
Thanks for putting that out. I'll see to it